docs/design/coreclr/jit/jit-disasm.md

```diff
@@ -0,0 +1,71 @@
+# JIT disassembly
+
+The JIT contains several configuration parameters that cause it to output textual disassembly of the generated code.
+Many of these parameters are available in the shipping product (a.k.a. the "Release" build flavor), as well as the internal
+Debug/Checked build flavors.
+
+These parameters include:
+* `DOTNET_JitDisasmSumary` - set to 1 to print a list of all JIT compiled functions.
+* `DOTNET_JitDisasm` - output disassembly for the specified functions. E.g., `DOTNET_JitDisasm=Main`, `DOTNET_JitDisasm=Main Test1 Test2`, `DOTNET_JitDisasm=*` (for all functions).
```

Comment:
I'd add a description for syntax used in JitDisasm - it raises a lot of questions in community. Or just more examples:
```
Method
Class:Method
Class:*
*Method*  (for top-level statements and nested methods because of name mangling)
```
Also, examples for generics and nested classes

Reply:
Also, can we add a link to this doc to jitconfigvalue.h next to JitDisasm? Presumably that's how some users find it

Reply:
Also, as a known issue, we might note that currently JitDisasm might print garbage when it's asked to print more than one method due to multiple compilations using printf at the same time

Reply:
Maybe I should just link to https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/jit/viewing-jit-dumps.md#specifying-method-names and possibly add more examples there? I don't want this document to be too much a duplicate of that one.

Reply:
I think linking to that section would make more sense.